lib/ui/src/components/panel/panel.tsx

```diff
@@ -59,14 +59,14 @@ const AddonPanel = React.memo<{
         <DesktopOnlyIconButton
           key="position"
           onClick={actions.togglePosition}
-          title="Change orientation"
+          title="Change addon orientation [D]"
         >
           <Icons icon={panelPosition === 'bottom' ? 'bottombar' : 'sidebaralt'} />
         </DesktopOnlyIconButton>
         <DesktopOnlyIconButton
           key="visibility"
           onClick={actions.toggleVisibility}
-          title="Hide addons"
+          title="Hide addons [A]"
```

Comment:
these are not hard coded, but rather user configurable!

Reply:
I assume to get them it's kinda like this?

```
import { Consumer } from '@storybook/api';

const Component = () => (
  <Consumer>
    {({ api }) => {
      const shortcuts = api.getShortcutKeys();
      return (
        <>
          <ComponentForFullScreen title={`Toggle fullscreen [${shortcuts.fullScreen}]`} />;
          <ComponentForAddonOrientation
            title={`Change addon orientation [${shortcuts.panelPosition}]`}
          />
          ;
          <ComponentForHidingAddons title={`Hide addons [${shortcuts.togglePanel}]`} />;
        </>
      );
    }}
  </Consumer>
);
```

Or is there another way to get them?

Reply:
Thanks for the feedback! I fixed this issue in my latest commit please let me what you think of my proposed solution.